libs/virtual-node/virtual-node-info/src/main/kotlin/net/corda/virtualnode/VirtualNodeInfo.kt

```diff
@@ -28,11 +28,17 @@ data class VirtualNodeInfo(
     val cryptoDmlConnectionId: UUID,
     /** HSM connection ID */
     val hsmConnectionId: UUID? = null,
+    /** Current state of the virtual node instance */
+    val state: String,
```

Comment:
```suggestion
    val state: String = DEFAULT_INITIAL_STATE,
```
I think if we do that, we can undo changes in many of the files where we had to assign this value specifically.